recipe/meta.yaml

```diff
@@ -1,48 +1,49 @@
 {% set name = "cadquery" %}
-{% set version = "1.2.5" %}
+{% set version = "2.2.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  - folder: dist
-    url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-    sha256: b2ddd927cb81bc9ad2332692d4ab0b4cb36581543463b64bd2d388b3709ea0ae
-    patches:
-      - osx_freecad_path.patch
   - folder: src
-    url: https://github.com/dcowden/cadquery/archive/v{{ version }}.tar.gz
-    sha256: 75eb6f987aeaf3d86197353c53dc2808a78fa9f2ade651f5751e96aa14b23829
+    url: https://github.com/CadQuery/cadquery/archive/refs/tags/{{ version }}.tar.gz
+    sha256: 9a71a3bb1185228f62b14810fe078489f4af61e02d9e2c683e68626ff9903dbc
 
 build:
   number: 0
   noarch: python
-  script: cd dist && python -m pip install --no-deps --ignore-installed .
+  script: cd src && python -m pip install --no-deps --ignore-requires-python --ignore-installed .
 
 requirements:
   host:
-    - python >=3.6,<3.7
+    - python >=3.8,<=3.11
+    - setuptools
     - pip
   run:
-    - python >=3.6,<3.7
-    - pyparsing
-    - freecad
+    - python >=3.8,<=3.11
+    - ocp 7.7.*
+    - pyparsing >=2.1.9
+    - ezdxf
+    - ipython
+    - typing_extensions
+    - nptyping 2.0.1
+    - nlopt
+    - multimethod >=1.7,<2.0
+    - casadi
 
 test:
-  source_files:
-    - src/tests
-    - src/runtests.py
   requires:
-    - mock
-  imports:
-    - cadquery
+    - pytest
+    - docutils
+    - path
```

Comment:
please add

```yaml
requires:
  - docutils
  - path
  - pip
  - pytest
commands:
  - pip check
  - pytest -vv
```

i'm also generally a fan of adding `pytest-cov` with a coverage baseline to help skip major dependency changes which might trigger conditional tests... this would also leave the `pytest` line more like

```bash
pytest -vv --cov cadquery --cov-branch --cov-report term-missing:skip-covered --no-cov-on-fail --cov-fail-under=XXX
```

Reply:
I'll add the line, but I don't really understand what you mean here. I.e. which conditional tests you refer to?

Reply:
Again, haven't been keeping up with the releases as you of course have. Sometimes a big drop in coverage signals that some optional (but important) dependency or build option has changed upstream in a way that needs the feedstock to be updated.

As it looks like cq is _much_ more soundly packaged/packageable now, we're in a good place to tighten up some of these things to help catch changes in an automated way in the future: of course, with more/better eyes on it, it should easier!